sunbeam-python/sunbeam/provider/maas/deployment.py

```diff
@@ -142,6 +146,11 @@ def infrastructure_model(self) -> str:
         """Return the infrastructure model name."""
         return "openstack-machines"
 
+    @property
+    def cluster_model(self) -> str:
```

Comment:
We need to think about the model names, they don't make sense as a whole.

We have:
- openstack
- openstack-machines
- sunbeam-cluster

While we already have a discussion about renaming `openstack`, I think we should think about the `sunbeam-cluster`, maybe renaming it to `openstack-infra` or something related? What do you think?


Of course, if the name becomes openstack-infra, we'll need to rename the property `infrastructure_model` to something else

Reply:
model name changed to `openstack-infra`
property `infrastructue_model` renamed to `openstack_machines_model`